altislife.sql

```diff
@@ -210,6 +212,12 @@ CREATE TABLE IF NOT EXISTS `wanted` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
 -- --------------------------------------------------------
+--
+-- Granting permissions to user `arma3`, created above
+--
+
+GRANT SELECT,UPDATE,INSERT,DELETE ON `altislife`.* TO 'arma3'@'localhost';
+GRANT EXECUTE ON altislife.* TO 'arma3'@'localhost';
```

Comment:
Why is `GRANT EXECUTE` on its own line?

Reply:
You're right, no need to duplicate this one.